src/aframe-streetmix-parsers.js

```diff
@@ -1007,14 +1001,12 @@ function processSegments(
 
   // create and center offset to center the street around global x position of 0
   var streetParentEl = createCenteredStreetElement(segments);
-  streetParentEl.classList.add('street-parent');
```

Comment:
Be careful, this class is checked in https://github.com/3DStreet/3dstreet/blob/2929a6d8b9aa5c1c2b3281fea2eb6659b26929e1/src/editor/components/components/AddLayerPanel/AddLayerPanel.component.jsx#L110-L111

Reply:
ok we'll change this later